### DISCUSSION TITLE
Remote sensor setup with Wireguard not appearing on grid and unable to send logs to Manager

### DISCUSSION THREAD

**Discussion by Guta-lang** [Security-Onion-Solutions/securityonion#8692] [Unsupported Versions]
I recently installed a remote sensor and setup WIREGUARD on the Sensor and Manager but unfortunately it doesn't appear on the grid neither is it sending traffic back to the manager. It's sending grafana data back as I can see it in the dashboard. Looking at the logs , one thing I'm able to pick from /opt/so/log/filebeat/filebeat.log are failed connections. 

{"log.level":"error","@timestamp":"2022-09-08T06:10:11.919Z","log.logger":"publisher_pipeline_output","log.origin":{"file.name":"pipeline/client_worker.go","file.line":150},"message":"Failed to connect to backoff(async(tcp://(Managername):5644)): dial tcp (manager-internal-IP):5644: connect: no route to host","service.name":"filebeat","ecs.version":"1.6.0"}

It looks like its trying to go back to the Manager's internal address which it wont be able to reach as its an REMOTE sensor and communication can only happen through the WIREGUARD interface , I can, however, ping the Manager's Wireguard address from the sensor, what can I change for the sensor  to point to the Manager's WIREGUARD  interface? 

Checking /opt/so/conf/sensoroni/sensoroni.JSON it shows the serverUrl as https://(manager-internal-IP)/sensoroniagents which could possibly be why it's not showing on the grid.

During setup of the sensor, I pointed the sensor to the Manager's IP, in this case the IP for the Wireguard interface.

**Comment by reyesj2** [comment]
Maybe [this](https://docs.securityonion.net/en/2.3/cloud-ami.html?highlight=ports#remote-sensor-setup) can point you in the right direction

**Comment by presianbg** [comment]
Just follow this:
https://github.com/Security-Onion-Solutions/securityonion/discussions/7128
